R/IDateTime.R

```diff
@@ -7,6 +7,10 @@ as.IDate = function(x, ...) UseMethod("as.IDate")
 
 as.IDate.default = function(x, ..., tz = attr(x, "tzone", exact=TRUE)) {
   if (is.null(tz)) tz = "UTC"
+  if (is.character(x)) {
+    # 4676 mimics for back-compatibility a similar patch applied to as.Date.character in r79119
```

Comment:
`# 4676` this will not be caught by a regex looking for issue number

Reply:
Good spot. Done in df8009d. Since it was just a comment tweak I did that directly without a PR